src/sources/tripadvisor/convert.rs

```diff
@@ -82,11 +123,47 @@ pub fn build_poi(
                 .map(ToString::to_string)
         })
         .unwrap_or_else(|| category.clone())
+        .replace(" ", "")
         .to_lowercase();
 
+    let cuisine = property
+        .cuisine
+        .inner
+        .into_iter()
+        .map(|item| {
+            let ta_cuisine = get_local_string(&["us".to_string()], &build_i18n_property(item.name))
+                .map(ToString::to_string)
+                .unwrap();
```

Comment:
That's the single `.unwrap()` I don't agree with: what should happen if `get_local_string` fails?

 - skip this field? we could use a `filter_map` on the iterator
 - panic as this shouldn't normally happen? use `.expect("...")` to explain what's going on

I'm being a bit picky, but usage of unwrap is generally discourage, even though there is a few places where it is ok to find one:

 - in tests
 - when you can prove that it will never fail, you built a good example of that a few lines bellow, however in this cases there is usually a construct that allows to remove the unwrap

My approach to that is to always put an `.expect("...")` when this is unlikely to fail, this helps a bit understanding the error in case it is triggered, but it mostly helps the reader to understand why this specific line may panic.

Reply:
Thank you for pointing this issue. I think I want the issue to be silent (not 100% sure) :  that's why I use `filter_map()` instead of `BuildError()` in the `fix review` commit.